### ISSUE TITLE
Factor out graph URI responsibility from gss-utils

### ISSUE THREAD

**Issue by ajtucker** [GSS-Cogs/pmd-jenkins-library#15]

**Comment by rossbowen** [comment]
We've recently made changes which embed graph information into CSVW metadata which, as explained in the linked issues above, I believe is universally agreed to not be the correct place for this sort of information.

Roughly, when dealing with large datasets, the approach being taken is to essentially run a pipeline multiple times, each time only processing a fraction of the data. On the first pass metadata and the dataset are also generated as part of our pipeline, on subsequent passes they are not. A new CSVW metadata file is created on each pass, and these metadata files contain triples specifying the graph which should be used to store those triples. The code which uploads data to PMD is now dependent on locating a graph triple in the RDF generated from the CSVW metadata. We've taken this approach to temporarily enable loading large data.

This issue isn't strictly related to the handling of large data, but I think if we can come up with a comfortable method for deriving PMD-bespoke metadata for large datasets, then simpler (smaller) cases should hopefully fall out neatly.

I'm trying to think about how third-party publishers may approach handling large data. To me, a natural approach would be,
- Chunk data into reasonably sized files. 
- Have a single CSVW metadata file with multiple `Table`s specified (one for each chunk), but a single `TableSchema` representing all chunks.

`csv2rdf` could then transform data on a CSV by CSV basis - I think this could even be performed in parallel.
From our side, we can use the knowledge of the data being held in separate CSV files to coin and assign a graph for the triples (naively, one graph per chunk'd CSV seems like it could work to me), and this could sit outside of `gssutils`.

I think a related scenario to consider would be a publisher who chunks their data month-by-month to provide their monthly update of statistics. A CSVW metadata file for a monthly update could contain a `Table` referencing the monthly CSV file, and point to the `TableSchema` used every other month.

I've added some [code to a repo](https://github.com/GSS-Cogs/pmd-metadata-from-csvw) which takes a CSVW metadata file which contains things like titles, descriptions etc. and strips out the information into a `.trig` file which attempts to take the form PMD requires. I see two benefits:
- Metadata such as titles, descriptions get to sit in the CSVW metadata file whereas they don't currently.
- PMD specific stuff, `pmdcat`, graphs etc. could be derived from the information provided in the metadata. A third party publisher only has to worry about getting their metadata correct and are spared any knowledge of the inner workings of PMD.

Very early days - I've not touched deriving graphs or any such things yet but I imagine that could be done in a similar fashion.
